absl/container/CMakeLists.txt

```diff
@@ -0,0 +1,77 @@
+#
+# Copyright 2017 The Abseil Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+file(GLOB CONTAINER_PUBLIC_HEADERS "*.h")
```

Comment:
Using `GLOB` to collect a list of source files is not recommended, since the GLOB is only re-run when a CMake file is changed, and not when a new header is added.

All of these GLOB's should be changed into explicit lists; which is exactly how the Bazel files do it.

Reply:
I like GLOB, specially when it avoids to maintain two separate list of source / headers file. But I do not have strong feeling about that. I will update it.

Reply:
Fixed